lib/parent-require.js

```diff
@@ -0,0 +1,9 @@
+module.exports = function (id) {
+  let parent = module.parent
+  for (; parent; parent = parent.parent) {
```

Comment:
see if it still works if you refactor it to not use `module.parent` anymore, as [per this comment](https://github.com/jaredhanson/node-parent-require/issues/4)